unischedule/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
     <uses-permission android:name="android.permission.USE_BIOMETRIC"/>
+    <uses-permission android:name="android.permission.SCHEDULE_EXACT_ALARM"/>
+    <uses-permission android:name="android.permission.USE_EXACT_ALARM"/>
```

Comment:
IDK if we have to ask for managing this permissions in the users Device when deploying the app

Reply:
Better safe than sorry!